berkshelf.gemspec

```diff
@@ -42,9 +42,9 @@ Gem::Specification.new do |s|
   s.add_dependency "octokit",              "~> 4.0"
   s.add_dependency "mixlib-archive",       ">= 1.1.4", "< 2.0" # needed for ruby 3.0 / Dir.chdir removal
   s.add_dependency "concurrent-ruby",      "~> 1.0"
-  if RUBY_VERSION.match?(/3.0/)
-    s.add_dependency "chef",                 "~> 17.0" # needed for --skip-syntax-check
-  elsif 
+  if RUBY_VERSION.match?(/3.3/)
```

Comment:
Chef 18 and 3.3 are by definition not going to be paired together unless installed as a gem. Chef 18 supports Ruby 3.0 for AIX and Ruby 3.1 for other platforms.